docs/run.md

```diff
@@ -128,12 +128,23 @@ https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage):
 
 4. Run LISA
 
-    Use above `<subscription id>` and `<private key file>` to replace in below command. It may take several minutes to complete.
+    Use above `<subscription id>` and `<private key file>` to replace
+    in commands below. They may take several minutes to complete,
+    depending on the complexity of the passed runbooks. A Marketplace
+    image *must* be provided (by its URN).
 
     ```bash
-    lisa -r ./microsoft/runbook/azure.yml -v subscription_id:<subscription id> -v "admin_private_key_file:<private key file>"
+    lisa -r ./microsoft/runbook/azure.yml -v subscription_id:<subscription id> -v "admin_private_key_file:<private key file>" -v "marketplace_image:Debian:debian-10:10:latest"
     ```
 
+	on Windows' cmd.exe or
+
+    ```bash
+    ./lisa.sh -r ./microsoft/runbook/azure.yml -v subscription_id:<subscription id> -v "admin_private_key_file:<private key file>" -v "marketplace_image:Debian:debian-10:10:latest"
+    ```
+
+	on a Linux shell.
+
```

Comment:
from https://github.com/glima/lisa/blob/main/docs/run.md, I feel the display a little bit strange.

Reply:
![image](https://user-images.githubusercontent.com/67689/114908236-c17ec080-9dd0-11eb-99fd-08d3e453eb84.png)
Can you point out what is exacly strange, please?